src/borg/archiver.py

```diff
@@ -4104,15 +4109,16 @@ def define_borg_mount(parser):
                                         '"borg help compression" command for details.')
         archive_group.add_argument('--recompress', metavar='MODE', dest='recompress', nargs='?',
                                    default='never', const='if-different', choices=('never', 'if-different', 'always'),
-                                   help='recompress data chunks according to ``--compression``. '
+                                   help='recompress data chunks according to `MODE` and ``--compression``. '
+                                        '(default) MODE `never`: '
+                                        'do not recompress.'
                                         'MODE `if-different`: '
                                         'recompress if current compression is with a different compression algorithm '
                                         '(the level is not considered). '
                                         'MODE `always`: '
                                         'recompress even if current compression is with the same compression algorithm '
                                         '(use this to change the compression level). '
-                                        'MODE `never` (default): '
-                                        'do not recompress.')
+                                        )
```

Comment:
src/borg/archiver.py:4121:41: E124 closing bracket does not match visual indentation